relation_engine_server/utils/config.py

```diff
@@ -45,10 +45,10 @@ def get_config():
             'release_id': os.path.join(spec_path, '.release_id'),
             'root': spec_path,
             'repo': spec_repo_path,
-            'schemas': spec_schemas_path,
+            'collections': collections_path,
             'stored_queries': stored_queries_path,
-            'views': spec_views_path,
-            'vertices': os.path.join(spec_schemas_path, 'vertices'),
+            'views': views_path,
+            'vertices': os.path.join(collections_path, 'vertices'),  # unused
```

Comment:
Let's remove this entry ("vertices")

We used to have a directory split of "vertices" and "edges", but later we removed that